src/adapter/src/catalog/open.rs

```diff
@@ -1761,6 +1767,24 @@ fn default_logging_config() -> ReplicaLogging {
         interval: Some(Duration::from_secs(1)),
     }
 }
+pub struct BuiltinBootstrapClusterSizes {
+    /// Size to default system_cluster init to
+    pub system_cluster: String,
+    /// Size to default introspection_cluster init to
+    pub introspection_cluster: String,
+}
```

Comment:
part of me thinks this should go in builtin.rs , but I don't really see it being used oustide of this one place, and I can't return `Result<String, mz_catalog::durable::CatalogError>` in get_size if I do that.

Reply:
I think it's fine here. `builtin.rs` is mostly things that are statically compiled into the binary, which doesn't exactly fit this struct. You could move it to `mz_catalog::config` and instead of having two String fields in `mz_catalog::config::StateConfig` you have a single field of type `BuiltinBootstrapClusterSizes`. Though, here is also fine IMO.